tests/framework/validators/ValidatorTest.php

```diff
@@ -214,4 +214,11 @@ public function testAddError()
         $errors = $m->getErrors('attr_msg_val');
         $this->assertEquals('attr_msg_val::abc::param_value', $errors[0]);
     }
+
+    public function testGetActiveValidatorsForSafeAttributes()
+    {
+        $model = $this->getTestModel();
+        $validators = $model->getActiveValidators('safe_attr');
+        $this->assertInstanceOf('yii\validators\NumberValidator', array_shift($validators));
```

Comment:
Sorry, I don't understand what you mean. 
Even if I will add  `'some_key' => ['!safe_attr', 'integer']` in `FakedValidationModel ` rules it won't help and I won't get this key from `getActiveValidators`. 
Could you please provide me some example. I will really appreciate it.